File: hm_world/src/main/java/com/ssafy/hm/dto/AttractionLine.java
package com.ssafy.hm.dto;

public class AttractionLine {
	private Integer lineId;
	private Integer attId;

	public AttractionLine() {
	}

	public AttractionLine(Integer lineId, Integer attId) {
		this.lineId = lineId;
		this.attId = attId;
	}

	public Integer getLineId() {
		return lineId;
	}

	public void setLineId(Integer lineId) {
		this.lineId = lineId;
	}

	public Integer getAttId() {
		return attId;
	}

	public void setAttId(Integer attId) {
		this.attId = attId;
	}


	@Override
	public String toString() {
		return "AttractionLine [lineId=" + lineId + ", attId=" + attId + "]";
	}
}

File: hm_world_app/app/src/main/java/com/ssafy/hm/data/model/AttractionLine.kt
package com.ssafy.hm.data.model

data class AttractionLine(
    val lineId: Int,
    val attId: Int,
    val members: List<AttractionLineMember> = emptyList()
)

data class AttractionLineMember(
    val id: Int,
    val lineId: Int,
    val userId: String
)
